src/index.html

```diff
@@ -8,16 +8,13 @@
```

Comment:
Good job! The code looks clean and concise. One suggestion I have is to consider adding more users to the usersTable array to test if the conditional statement works for multiple users. Other than that, keep up the great work!